### ISSUE TITLE
Package conflict with `nvidia-utils`

### ISSUE THREAD

**Issue by jcstryker** [erin-allison/nvidia-merged-arch#5]
I am trying to replace the current latest set of nvidia packaged in the arch repos using this command
```
paru -U ./lib32-nvidia-merged-utils-460.73.01-7-x86_64.pkg.tar.xz ./lib32-opencl-nvidia-merged-460.73.01-7-x86_64.pkg.tar.xz ./nvidia-merged-dkms-460.73.01-7-x86_64.pkg.tar.xz ./nvidia-merged-settings-460.73.01-7-x86_64.pkg.tar.xz ./nvidia-merged-utils-460.73.01-7-x86_64.pkg.tar.xz ./opencl-nvidia-merged-460.73.01-7-x86_64.pkg.tar.xz
```

This results in a package conflict shown below
```
loading packages...
resolving dependencies...
looking for conflicting packages...
:: nvidia-merged-utils and nvidia-utils are in conflict (nvidia-libgl). Remove nvidia-utils? [y/N] y
:: lib32-nvidia-merged-utils and lib32-nvidia-utils are in conflict (lib32-nvidia-libgl). Remove lib32-nvidia-utils? [y/N] y
error: failed to prepare transaction (could not satisfy dependencies)
:: removing nvidia-utils breaks dependency 'nvidia-utils' required by nvidia-prime
:: removing nvidia-utils breaks dependency 'nvidia-utils' required by nvidia-settings
:: removing nvidia-utils breaks dependency 'nvidia-utils' required by python-py3nvml
```

I think this is because the PKGBUILD does not configure `nvidia-merged-utils` to provide `nvidia-utils`.

I noticed that we do provide this for `lib32-nvidia-merged-utils`, is there a reason we dont provide this behavior with the non `lib32` package?

**Comment by erin-allison** [comment]
Nope, it's just an issue that had not come up yet.

**Comment by erin-allison** [comment]
@jcstryker This should be fixed as of b19dabc. Pull the latest master and give it a try now, if you would.
